### ISSUE TITLE
Some man files ship in the wrong package in Debian packages

### ISSUE THREAD

**Issue by peternewman** [OpenLightingProject/ola#1276]
The RDM responder tester man files all ship in the main ola package:
https://packages.debian.org/stretch/amd64/ola/filelist
```
/usr/share/man/man1/rdm_model_collector.py.1.gz
/usr/share/man/man1/rdm_responder_test.py.1.gz
/usr/share/man/man1/rdm_test_server.py.1.gz
```

While their binaries (well Python scripts) ship in the ola-rdm-tests package:
https://packages.debian.org/stretch/all/ola-rdm-tests/filelist
```
/usr/bin/rdm_model_collector.py
/usr/bin/rdm_responder_test.py
/usr/bin/rdm_test_server.py
```

@yoe I'm guessing ideally this wants some collaboration on our end too, hence raising it here.

Currently we just generate dist_man_MANS:
https://github.com/OpenLightingProject/ola/blob/master/man/Makefile.mk#L1

If we made two separate variables, and then combined them to keep backwards compatibility, could dpkg-deb make use of them to avoid having to store the metadata about what goes where separately elsewhere?

**Comment by yoe** [comment]
The Debian packaging tools really only check the files under `debian` for deciding which file goes where; in this case, they check `debian/ola.install` and `debian/ola-rdm-test.install`. Adding a variable doesn't help, since those files are just plain text files; and while we *could*, in theory, make them shell scripts, that doesn't buy us much of anything in this case.

I'm guessing that when the `.install` files were written, there weren't that many man pages yet, and then just installing them all in the same package is much easier. Now that isn't correct anymore, so we have to fix it.

I'll look at it tonight.

**Comment by peternewman** [comment]
Thanks @yoe . I was thinking of the debian/ola.dirs file instead for some reason.

That looks like it should be a fairly easy fix, then we just need to look at Lintian carefully to check we've not regressed, or missed out an existing man page.

**Comment by peternewman** [comment]
Can you take look at #1279 please @yoe ?

**Comment by peternewman** [comment]
Closed via #1279 .
